include/GLFW/glfw3.h

```diff
@@ -2107,6 +2142,9 @@ GLFWAPI void glfwGetWindowSize(GLFWwindow* window, int* width, int* height);
  *  @remark If you set size limits and an aspect ratio that conflict, the
  *  results are undefined.
  *
+ *  @remark @wayland The size limits will not be applied until the window is
+ *  actually resized, either by the user or by the compositor.
```

Comment:
This is a bug. If necessary, the limits [should be applied manually](https://github.com/glfw/glfw/blob/3.2.1/src/win32_window.c#L1151).